From 530ef7f26ed041ab1b6655f2cdc79f3af97fc50b Mon Sep 17 00:00:00 2001
From: "VMware, Inc" <>
Date: Thu, 27 Dec 2012 11:51:12 -0800
Subject: [PATCH] OVT: do not build vmblock on Linux kernels 3.0+

The driver fails to build on newer kernels and we recommend using
the FUSE-based solution (Vmware Tools use vmblock-fuse on kernels
starting with 2.6.32), so let's stop building it.

Signed-off-by: Dmitry Torokhov <dtor@vmware.com>
---
 open-vm-tools/configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/configure.ac
+++ b/configure.ac
@@ -943,7 +943,11 @@ if test "$os" = "linux"; then
    LIBVMTOOLS_LIBADD="$LIBVMTOOLS_LIBADD -lrt"
 
    MODULES="$MODULES vmsync vmci vsock"
-   MODULES="$MODULES vmxnet vmblock vmhgfs"
+   MODULES="$MODULES vmxnet vmhgfs"
+   if test "$osVersion" -lt 300000; then
+       MODULES="$MODULES vmblock vmsync"
+   fi
+
    buildHgfsmounter=yes
 fi
 
